apps/src/templates/manageStudents/ManageStudentsTable.jsx

```diff
@@ -210,6 +210,41 @@ class ManageStudentsTable extends Component {
 
   // Cell formatters.
 
+  passwordHeaderFormatter = () => {
+    const {loginType} = this.props;
+    const passwordLabels = {};
```

Comment:
Nice to see you using hashmaps for this sort of thing!

nit - this is a totally subjective, nonblocking comment. I think this code looks great as is and I'd like it without any changes, but I think it'd be a little bit more readable if you initialized the values for passwordLabels and passwordTooltips for each key side by side, like this:

```
const passwordLabels = {}
const passwordTooltips = {}

passwordLabels[SectionLoginType.picture] = i18n.picturePassword()
passwordTooltips[SectionLoginType.picture] = i18n.editSectionLoginTypePicDesc()

passwordLabels[SectionLoginType.word] = i18n.secretWords()
passwordTooltips[SectionLoginType.word] = i18n.editSectionLoginTypeWordDesc()

... etc ...
```

This makes it slightly easier to see that passwordTooltips and passwordLabels each have a value for each SectionLoginType, and to edit multiple values for any individual key if you ever have the need to. It'd also make it easier to add new metadata values for a new key, since doing this would require less scrolling - everything would just go into one contiguous block of new code rather than in several sections of code.

 In this case since there are only three SectionLoginTypes and only two categories of metadata you are defining for each what you have is about as readable, but if there were more loginTypes or more categories of metadata I think grouping the metadata values by key would be a bit more noticeably clear. I think its always a good practice to expect code to become more complicated over time, and to code accordingly, and to expect that we may add new categories or logintypes later.

Again, this is all subjective and I think what you have looks great - just thought I'd share these thoughts.